src/FretboardSVG.module.css

```
@@ -266,14 +227,41 @@
+  text-align: center;
+  font-family: var(--font-sans);
+  font-size: 0.9rem;
+  font-weight: 500;
+  color: rgb(191 204 216 / 0.72);
+  user-select: none;
+  -webkit-user-select: none;
+}
+
+/* Responsive Overrides */
+:global(.app-container[data-layout-tier="desktop"]) .fret-number {
+}
```

Stylelint (17.7.0)

[error] Empty block (block-no-empty)

(block-no-empty)